crates/astria-eyre/Cargo.toml

```diff
@@ -8,8 +8,12 @@ edition = "2021"
 [dependencies]
 eyre = "0.6"
 itoa = "1.0.10"
+anyhow = { version = "1", optional = true }
 
 [dev-dependencies]
 tracing = { workspace = true }
 tokio-test = { workspace = true }
 tracing-subscriber = "0.3.18"
+
+[features]
+anyhow_support = ["anyhow"]
```

Comment:
This is not necessary. Remove the extra feature. Because rust dependencies and features (unfortunately) share the same namespace, you can activate it by using `cargo build --features anyhow`